app/views/reports/regions/show.html.erb

```diff
@@ -272,24 +272,214 @@
+            </colgrup>
+            <thead>
+              <tr class="sorts" data-sort-method="thead">
+                <th class="row-label sort-label sort-label-small ta-left">
+                  Name
+                </th>
+                <th class="row-label sort-label sort-label-small ta-left" data-sort-method="number" data-sort-default>
+                  BP &lt;140/90
+                </th>
+                <th class="row-label sort-label sort-label-small ta-left" data-sort-method="number">
+                  Number of patients
+                </th>
+              </tr>
+            </thead>
+            <tbody>
+              <% @region.facilities.order(:name).each do |facility| %>
+                <tr>
+                  <td class="ta-left">
+                    <%= link_to(reports_region_path(facility.region_slug)) do %>
+                      <%= facility.name %>
+                    <% end %>
+                  </td>
+                  <td class="ta-left">
+                    PENDING
+                  </td>
+                  <td class="ta-left">
+                    PENDING
+                  </td>
+                </tr>
+              <% end %>
+            </tbody>
+          </table>
+        </div>
+      </div>
+    </div>
+    <div class="d-lg-flex w-lg-50 pl-lg-2">
+      <div class="card d-lg-flex fd-lg-column justify-lg-between h-lg-full w-lg-full mt-lg-0">
+        <div class="mb-2">
+          <h3 class="c-black">
+            No BP measure
+          </h3>
+        </div>
+        <div class="table-responsive">
+          <table id="no-bp-measure-table" class="table-compact">
+            <colgroup>
+              <col>
+              <col class="table-divider">
+              <col class="table-divider">
+              <col class="table-divider">
+            </colgrup>
+            <thead>
+              <tr class="sorts" data-sort-method="thead">
+                <th class="row-label sort-label sort-label-small ta-left">
+                  Name
+                </th>
+                <th class="row-label sort-label sort-label-small ta-left" data-sort-method="number" data-sort-default>
+                  Missed visit
+                </th>
+                <th class="row-label sort-label sort-label-small ta-left" data-sort-method="number">
+                  Lost to follow-up
+                </th>
+                <th class="row-label sort-label sort-label-small ta-left" data-sort-method="number">
+                  Visited but no BP
+                </th>
+              </tr>
+            </thead>
+            <tbody>
+              <% @region.facilities.order(:name).each do |facility| %>
+                <tr>
+                  <td class="ta-left">
+                    <%= link_to(reports_region_path(facility.region_slug)) do %>
+                      <%= facility.name %>
+                    <% end %>
+                  </td>
+                  <td class="ta-left">
+                    PENDING
```

Comment:
These `PENDING` here are intentional?

Reply:
They're intentional and addressed in (this PR)[https://github.com/simpledotorg/simple-server/pull/1189]